ddpui/ddpairbyte/airbytehelpers.py

```
@@ -646,6 +652,43 @@ def update_connection(org: Org, connection_id: str, payload: AirbyteConnectionUp
+                        "config": {
+                            "tasks": [task_config.to_json()],
+                            "org_slug": org.slug,
+                        }
+                    }
+                    prefect_service.update_dataflow_v1(
+                        dot.dataflow.deployment_id,
+                        PrefectDataFlowUpdateSchema3(
+                            deployment_params=new_params, cron=dot.dataflow.cron
+                        ),
+                    )
+                except Exception as err:  # pylint: disable=broad-exception-caught
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)